### DISCUSSION TITLE
Search POIs without an external SQLite database

### DISCUSSION THREAD

**Discussion by emv412** [mapsforge/vtm#1132] [Q&A] [answered]
Is it possible to search POIs on a mapsforge's binary map fornat (.map extension) without using an external POI SQLite database? In the specification (https://github.com/mapsforge/mapsforge/blob/master/docs/Specification-Binary-Map-File.md) I read that POI data is embedded into the file.

Thanks!

**Comment by devemux86** [comment, marked as answer]
This is not possible.

Map files are not searchable databases, they are optimized for map rendering and contain only the rendering data.
